src/components/comment-form/comment-form.twig

```diff
@@ -1,11 +1,18 @@
 <form
   class="c-comment-form{% if class %} {{class}}{% endif %}"
+  {% if heading_id %}aria-labelledby="{{ heading_id }}"{% endif %}
```

Comment:
This was added because I found out forms don't get a role of `form` without an accessible name:

https://github.com/testing-library/dom-testing-library/issues/937